queries/ledger/highlights.scm

```diff
@@ -1,16 +1,26 @@
 [
+    (block_comment)
     (comment)
     (note)
+    (test)
 ] @comment
 
 [
     (date)
+    (effective_date)
+    (time)
```

Comment:
Are these numeric? If not, they should be `@constant`.

Reply:
Yes they are, e.g. `2023-04-23`. They especially feel numeric with something like [tpope/vim-speeddating](https://github.com/tpope/vim-speeddating)

Reply:
`2023-04-23` is not entirely numeric.

Reply:
This is in the context of ledger, which is not a programming language but a language for describing financial transactions.
In this context, it *feels* much more like a numeral than a constant, string, etc.

Reply:
I guess @string.special could work here too given the description in CONTRIBUTING.md

Reply:
Given that dates generally may include month and day names `@string.special` seems very fitting, especially as [`CONTRIBUTING.md` §Literals](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#literals) calls dates out explicitly